suite-native/app/src/App.tsx

```diff
@@ -128,4 +129,4 @@ const PureApp = () => (
     </GestureHandlerRootView>
 );
 
-export const App = Sentry.wrap(PureApp);
+export const App = isDebugEnv() ? Sentry.wrap(PureApp) : PureApp;
```

Comment:
This seems inverted to me. And it won't stop Sentry.wrap from being called before Sentry.init, will it?

Reply:
Hmm, Sentry.init is being called in the useEffect, so you are probably right. But at least it wont bother us on debug while developing the app.